client/src/components/modal-forms/NewParticipantForm.js

```diff
@@ -119,6 +135,62 @@ export const NewParticipantForm = ({ submissionCallback, onClose, sites }) => {
                 }))
                 .sort((a, b) => a.label.localeCompare(b.label))}
             />
+            <Field
+              name='driversLicense'
+              component={RenderSelectField}
+              label='* Class 5 Drivers License'
+              options={[
+                { value: 'yes', label: 'Yes' },
+                { value: 'no', label: 'No' },
+                { value: 'unknown', label: 'Unknown' },
+              ]}
+            />
+            <Field
+              name='indigenous'
+              component={RenderSelectField}
+              label='Self-identifies as First Nation, Métis, Inuk (Inuit) or Urban Indigenous?'
+              options={[
+                { value: 'yes', label: 'Yes' },
+                { value: 'no', label: 'No' },
+                { value: 'preferNotTOSay', label: 'Prefer not to say' },
+              ]}
+            />
+            <Field
+              name='educationalRequirements'
+              component={RenderSelectField}
+              label='Meets educational requirements'
+              options={[
+                { value: 'yes', label: 'Yes' },
+                { value: 'no', label: 'No' },
+                { value: 'dontknow', label: 'Does not know' },
+              ]}
+            />
+            <Field
+              name='currentOrMostRecentIndustry'
+              component={RenderTextField}
+              label='What industry did they most recently work in?'
```

Comment:
don't the users respond about themselves? I wonder if it should be 'did you'.

Reply:
This is when employers / HAs are adding non portal hires, so they would be appropriate!